src/main/R/FDR_plot.R

```diff
@@ -1,26 +1,39 @@
 library(ggplot2)
 
 # load FDR and CGD sets
-fdr <- read.table("/Users/joeri/github/rvcf/src/test/resources/bundle_r1.0/FDR_allGenes.tsv", sep="\t", header=T)
-cgd <- read.table("/Users/joeri/github/rvcf/src/test/resources/GenesInheritance.tsv", sep="\t", header=T)
+fdr <- read.table("/Users/joeri/github/rvcf/src/test/resources/bundle_r1.0/FDR_allGenes_r1.0.tsv", sep="\t", header=T)
+cgd <- read.table("/Users/joeri/github/rvcf/src/test/resources/GenesInheritance31aug2016.tsv", sep="\t", header=T)
+
```

Comment:
hardcoded paths, not sure how to fix in R, can those be relative to the project?